core/src/providers/mistral.rs

```diff
@@ -746,6 +781,8 @@ impl MistralAILLM {
         body.reader().read_to_end(&mut b)?;
         let c: &[u8] = &b;
 
+        // println!("MistralAI response: {}", String::from_utf8_lossy(c));
```

Comment:
nit

Reply:
They are very convenient in favor of keeping them